README.md

```diff
@@ -39,7 +39,7 @@ This will add `/graphql` and `/graphiql` endpoints to your app.
  * `executor`: The `Executor` that you want to use to execute queries.
  * `graphiql`: If `True`, may present [GraphiQL](https://github.com/graphql/graphiql) when loaded directly from a browser (a useful tool for debugging and exploration).
  * `graphiql_template`: Inject a Jinja template string to customize GraphiQL.
- * `batch`: Set the GraphQL view as batch (for using in [Apollo-Client](http://dev.apollodata.com/core/network.html#query-batching) or [ReactRelayNetworkLayer](https://github.com/nodkz/react-relay-network-layer))
+ * `batch`: Set the GraphQL view as batch (for using in [Apollo-Client]https://www.apollographql.com/docs/react/networking/network-layer/#query-batching) or [ReactRelayNetworkLayer](https://github.com/nodkz/react-relay-network-layer))
```

Comment:
You missed an opening parentheses before the url.

Reply:
Fixed, thank you